app/models/assignment.rb

```diff
@@ -427,9 +429,12 @@ def add_csv_group(row)
     # shouldn't happen anyway, because the lookup earlier should prevent
     # repo collisions e.g. when uploading the same CSV file twice.
     group.save
+    m_logger.log("Created Group with name:'#{group.group_name}' and repo name: '#{group.repo_name}' 
+                  with error:'#{group.errors[:base]}'")
     unless group.errors[:base].blank?
+      m_logger.log("Collision detected of '#{group.errors[:base]}'", MarkusLogger::ERROR)
       collision_error = I18n.t('csv.repo_collision_warning',
-                          { repo_name: group.errors.on_base,
+                          { repo_name: group.errors.get(:base),
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.